python/examples/tchannel_handler_server.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+
+import argparse
+import logging
+import sys
+
+import tornado.ioloop
+import tornado.web
+from tchannel.tornado import TChannel
```

Comment:
Feel free to replace the existing tornado example server with this stuff, it's nice.